request-management-api/request_api/models/FOIRawRequests.py

```diff
@@ -1,14 +1,19 @@
+from enum import unique
+import re
+
+from sqlalchemy.sql.expression import distinct
 from .db import  db, ma
 from sqlalchemy.dialects.postgresql import JSON, UUID
 from .default_method_result import DefaultMethodResult
 from datetime import datetime
+from sqlalchemy import insert, and_
 
 class FOIRawRequest(db.Model):
     # Name of the table in our database
     __tablename__ = 'FOIRawRequests' 
     # Defining the columns
     requestid = db.Column(db.Integer, primary_key=True,autoincrement=True)
-    version = db.Column(db.Integer, primary_key=True)
+    version = db.Column(db.Integer, primary_key=True,nullable=False)
```

Comment:
Is this gonna work with existing data in test/dev? Is data null now?

Reply:
@acoard _ Yes.